src/stack.jl

```diff
@@ -40,7 +40,7 @@ Base.eltype(::Type{Stack{T}}) where T = T
 
 Get a top item from a stack
 """
-top(s::Stack) = back(s.store)
+top(s::Stack) = last(s.store)
```

Comment:
we should replace and deprecate this too.

```suggestion
first(s::Stack) = last(s.store)
```